openapi.yaml

```diff
@@ -0,0 +1,675 @@
+
+    OriginatingIdentity:
+      name: X-Broker-API-Originating-Identity
+      in: header
+      description: identity of the user that initiated the request from the Platform
+      schema:
+        type: string
+
+  schemas:
+    Catalog:
+      type: object
+      properties:
+        services:
+          type: array
+          items:
+            $ref: '#/components/schemas/Service'
+
+    Service:
+      required:
+        - name
+        - id
+        - description
+        - bindable
+        - plans
+      properties:
+        name:
+          type: string
+        id:
+          type: string
+        description:
+          type: string
+        tags:
+          type: array
+          items:
+            type: string
+        requires:
+          type: array
+          items:
+            type: string
+            enum:
+              - syslog_drain
+              - route_forwarding
+              - volume_mount
+        bindable:
+          type: boolean
+        metadata:
+          $ref: '#/components/schemas/Object'
+        dashboard_client:
+          $ref: '#/components/schemas/DashboardClient'
+        plan_updateable:
+          type: boolean
+        plans:
+          type: array
+          items:
+            $ref: '#/components/schemas/Plan'
+
+    DashboardClient:
+      properties:
+        id:
+            type: string
+        secret:
+            type: string
+        redirect_uri:
+            type: string
+
+    Plan:
+      required:
+        - id
+        - name
+        - description
+      properties:
+        id:
+          type: string
+        name:
+          type: string
+        description:
+          type: string
+        metadata:
+          $ref: '#/components/schemas/Object'
+        free:
+          type: boolean
+          default: true
+        bindable:
+          type: boolean
+        schemas:
+          $ref: '#/components/schemas/SchemasObject'
+
+    SchemasObject:
+      properties:
+        service_instance:
+          $ref: '#/components/schemas/ServiceInstanceSchemaObject'
+        service_binding:
+          $ref: '#/components/schemas/ServiceBindingSchemaObject'
+
+    ServiceInstanceSchemaObject:
+      properties:
+        create:
+          $ref: '#/components/schemas/SchemaParameters'
+        update:
+          $ref: '#/components/schemas/SchemaParameters'
+
+    ServiceBindingSchemaObject:
+      properties:
+        create:
+          $ref: '#/components/schemas/SchemaParameters'
+
+    SchemaParameters:
+      properties:
+        parameters: 
+          $ref: '#/components/schemas/JSONSchemaObject'
+
+    JSONSchemaObject:
+      type: object
+
+    ServiceInstanceProvisionRequest:
+      required:
+        - service_id
+        - plan_id
+        - organization_guid
+        - space_guid
+      properties:
+        service_id:
+          type: string
+        plan_id:
+          type: string
+        context:
+          $ref: '#/components/schemas/Object'
```

Comment:
Is there an easy way to link to the [Context Conventions](https://github.com/openservicebrokerapi/servicebroker/blob/master/profile.md#context-object) here? Maybe via a `description`?

Reply:
Did you spot this one too @n3wscott ?